keanu-project/src/test/java/io/improbable/keanu/tensor/TensorTestHelper.java

```diff
@@ -0,0 +1,21 @@
+package io.improbable.keanu.tensor;
+
+import static org.junit.Assert.assertEquals;
+
+public class TensorTestHelper {
+
+    public static void doesDownRankOnSliceRank3To2(Tensor x) {
```

Comment:
Do we need all these? Can we just have a `doesDownRankOnSlice(Tensor x, int expectedRank)`?